Functions.Templates/Templates/SqlInputBinding-CSharp/SqlInputBindingCSharp.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Logging;
+
+namespace Company.Function
+{
+    public static class SqlInputBindingCSharp
+    {
+        // Visit https://github.com/Azure/azure-functions-sql-extension#Input-Binding-Tutorial to learn how to use this input binding
```

Comment:
Is anything tracking changes to readme to avoid broken links in the future?

Reply:
Linking to our docs aka.ms/sqlbindingsinput is probably a better option here - I'll update that.